assets/component-facets.css

```diff
@@ -1040,8 +1040,9 @@ input.mobile-facets__checkbox {
     margin-bottom: 1.5rem;
   }
 
-  .facets-vertical .facet-filters.sorting.no-js {
+  .no-js .facets-vertical .facet-filters.sorting {
     padding-left: 0;
+    flex-direction: column;
   }
```

Comment:
I chose to move `no-js` to the form for now. Even though I couldn't access it with a screenreader, it didn't have `display: none`.